### PR TITLE
docs: compact config

### PR DESCRIPTION
**Pull request by ZeRego**
Closes: #3701

### CODE REVIEW

**Review by rephus — APPROVED** [review]

File: docs/docs/guides/formatting-your-fields.mdx [hunk 1]
```diff
@@ -204,8 +204,46 @@ models:
...
+version: 2
+models:
+- name: sales
+  columns:
+    - name: revenue
+      meta:
+        dimension:
+          compact: millions
+```
+
```
Comment (TuringLovesDeathMetal) [inline docs/docs/guides/formatting-your-fields.mdx:229]: Can you add an example of what the output would look like here? 

E.g. a screenshot with a compact number

File: docs/docs/references/dimensions.mdx [hunk 2]
```diff
@@ -81,18 +81,28 @@ models:
...
+| format                            | No       | string                                                                                                                                            | This option will format the output value on the result table and CSV export. Currently supports one of the following: `['km', 'mi', 'usd', 'gbp', 'eur', 'percent']`                                                                                       |
+| [compact](#compact)               | No       | string                                                                                                                                            | This option will compact the number value on the result table and CSV export. Currently supports one of the following: `['thousands', 'millions', 'billions', 'trillions']`                                                                                |
```
Comment (TuringLovesDeathMetal) [inline docs/docs/references/dimensions.mdx:94]:
```suggestion
| [compact](#compact-values)               | No       | string                                                                                                                                            | This option will compact the number value (e.g. 1,500 to 1.5K). Currently supports one of the following: `['thousands', 'millions', 'billions', 'trillions']`                                                                                |
```

File: docs/docs/references/dimensions.mdx [hunk 3]
```diff
@@ -81,18 +81,28 @@ models:
...
+| group_label                       | No       | string                                                                                                                                            | If you set this property, the dimension will be grouped in the sidebar with other dimensions with the same group label.                                                                                                                                    |
+| [urls](#urls)                     | No       | Array of { url, label }                                                                                                                           | Adding urls to a dimension allows your users to click dimension values in the UI and take actions, like opening an external tool with a url, or open at a website. You can use liquid templates to customise the link based on the value of the dimension. |
+
+## Compact values
+
```
Comment (TuringLovesDeathMetal) [inline docs/docs/references/dimensions.mdx:99]:
```suggestion
You can compact values in your YAML. For example, if I wanted all of my revenue values to be shown in thousands (e.g. `1,500` appears as `1.5K`), then I would write something like this in my .yml:

```yaml
version: 2
models:
- name: sales
  columns:
    - name: revenue
      meta:
        dimension:
            compact: thousands # I can also use 'K' 
```
 
 We currently support these compact value formats:
```

File: docs/docs/references/metrics.mdx [hunk 4]
```diff
@@ -457,9 +458,18 @@ The list of fields must be made of dimension names (no metrics) from the base ta
 
 The order that the fields are listed in `show_underlying_values` is the order that they'll appear in on the `view underlying data` table.
 
+## Compact values
```
Comment (TuringLovesDeathMetal) [inline docs/docs/references/metrics.mdx:461]:
```suggestion
## Compact values

You can compact values in your YAML. For example, if I wanted all of my total revenue values to be shown in thousands (e.g. `1,500` appears as `1.5K`), then I would write something like this in my .yml:

```yaml
version: 2
models:
- name: sales
  columns:
    - name: revenue
      meta:
        metrics:
          total_revenue:
            type: sum
            compact: thousands
```

**Review by TuringLovesDeathMetal — COMMENTED** [review]
just a few additions :) looks great otherwise!

**Pull request merged**